build.gradle.kts

```diff
@@ -70,14 +69,13 @@ nexusPublishing {
 
 java {
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of(jdkVersion))
+        languageVersion.set(of(8))
```

Comment:
optional: we may want to avoid the static import here so that we can use recipes to modify these if we desire to.